test_sets/Defects/Defects_releases_4_x_and_5_x.txt

```diff
@@ -0,0 +1,121 @@
+;./test_scripts/Defects/5_0/972_SDL_should_respond_IGNORED_with_correct_result_code_for_UnSubscribeVehicleData_in_case_vi_interface_is_not_available.lua https://github.com/smartdevicelink/sdl_core/issues/3205
+;./test_scripts/Defects/5_0/2457_Needed_fixes_sending_of_UpdateDeviceList_notification_on_device_connect.lua https://github.com/smartdevicelink/sdl_atf_test_scripts/issues/2365
+./test_scripts/Defects/5_0/1031_SDL_transfer_OnKeyboardInput_notification_to_not_active_App_when_there_is_no_active_PerformInteraction_KEYBOARD.lua
+./test_scripts/Defects/5_0/2480_Vefiry_that_SDL_sends_OnDD_notification_once_upon_registration.lua
+./test_scripts/Defects/5_0/1035_SDL_doesnt_send_REJECTED_code_to_mobile_app_when_activating_app_from_HMI_with_activate.lua
+./test_scripts/Defects/5_0/1218_Mandatory_parameter_spaceAvailable_is_not_provided_in_response_for_PutFile_request.lua
+;./test_scripts/Defects/5_0/1032_CreateInteractionChoiceSet_core_successfully_creates_choice_set_with_duplicate_vrCommands_menuName_inside_it.lua https://github.com/smartdevicelink/sdl_atf_test_scripts/issues/2330
+./test_scripts/Defects/5_0/2443_SDL_does_not_check_for_non_manadatory_parameters.lua
+./test_scripts/Defects/5_0/964_SDL_sends_to_mobile_APPLICATION_NOT_REGISTRED_in_setAppIcon_responce_if_HMI_responds_with_INVALID_DATA.lua
+./test_scripts/Defects/5_0/1030_API_SDL_sends_OnAppInterfaceUnregistered(DRIVER_DISTRACTION_VIOLATION)_to_app_when_receives_OnExitApplication(DRIVER_DISTRACTION_VIOLATION)_from_HMI.lua
+;./test_scripts/Defects/5_0/1866_SDL_does_not_respond_to_HMI_request_with_missed_mandatory_params.lua https://github.com/smartdevicelink/sdl_atf_test_scripts/issues/2329
+;./test_scripts/Defects/5_0/1867_SDL_does_not_respond_to_HMI_request_with_wrong_type_of_params.lua https://github.com/smartdevicelink/sdl_atf_test_scripts/issues/2329
+;./test_scripts/Defects/5_0/1868_SDL_does_not_respond_to_HMI_request_with_empty_string_parameter.lua https://github.com/smartdevicelink/sdl_atf_test_scripts/issues/2329
+./test_scripts/Defects/5_1/2862_SDL_sends_UnsubscribeWayPoint_request_to_HMI.lua
+./test_scripts/Defects/5_1/2874_SDL_sends_system_request_with_correlationid_-1_and_INAVLID_DATA.lua
+;-----------------------------------------------------------------------------------------------------
+;-----------------------------------------------------------------------------------------------------
+;Stream:
+;-----------------------------------------------------------------------------------------------------
+./test_scripts/Defects/4_5/1912_Video_service_start_via_2_protocol.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_1_SDL_does_not_respond_NACK_Navi_Type.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_2_SDL_does_not_respond_NACK_Audio_Type.lua
+./test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_3_SDL_respond_NACK_from_protected_to_not_protected.lua
```

Comment:
[FAIL] (2622 ms)
 HMI notification Navigation.OnVideoDataStreaming: Times: The most allowed occurences boundary exceed
 HMI notification Navigation.OnVideoDataStreaming: ValidIf: 
params.available: expected: true, actual value: false
params.available: expected: true, actual value: false
[00:47:01,624] Start_video_service_via_2_protocol_with_expectation_of_StartServiceNACK